src/serial.c

```diff
@@ -97,14 +97,28 @@ static void *serial_thread(serial_dev_t *s)
     return NULL;
 }
 
+#define TERMINAL_ESCAPE_CHAR 0x01
+#define TERMINAL_TERMINATE_CHAR 'x'
+
 void serial_console(serial_dev_t *s)
 {
     struct serial_dev_priv *priv = (struct serial_dev_priv *) s->priv;
+    static bool escaped = false;
 
     while (!fifo_is_full(&priv->rx_buf) && serial_readable(s, 0)) {
         char c;
         if (read(s->infd, &c, 1) == -1)
             break;
+        if (escaped && c == TERMINAL_TERMINATE_CHAR) {
+            /* Terminate */
+            fprintf(stderr, "\nTerminated\n");
```

Comment:
I am not if it makes sense to dump this messages.

Reply:
I changed it to `fprintf(stderr, "\n");`.
The newline character is inserted just before kvm-host is terminated.  This ensures that the shell prompt starts at the beginning of a line.